go.mod

```diff
@@ -50,3 +54,15 @@ replace (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8sclusterreceiver v0.0.0-00010101000000-000000000000 => github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8sclusterreceiver v0.15.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.0.0-00010101000000-000000000000 => github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.15.0
 )
+
+// each of these is required for the smartagentreceiver
```

Comment:
I believe only for grpc (currently 1.29.1 vs 1.34.0 in the collector) since that requires ongoing changes in etcd: https://github.com/etcd-io/etcd/pull/12398 (and more likely https://github.com/etcd-io/etcd/pull/12597)